### DISCUSSION TITLE
Episode 212: No More AI and Episode 211

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#111] [Episode]
Episode Link: https://adspthepodcast.com/2024/12/13/Episode-212.html

Feel free to write comments or ask questions 🙂

**Comment by koenpoppe** [comment]
As of March this year, Apple is providing full transcripts for its Podcasts ([source]). This is what they made from the section discussing this:
> Yeah, but the problem is, is that it doesn't, I have to go and fix it because we're going to dice this up, right? Whereas what I want to be able to do is like feed it the finished MP3. And that's the other thing is some people have requested a video and for us to upload it to YouTube of us chatting.
> 
> But once again, like I edit the audio and like having to edit the video would like increase the amount of work by a factor of like four or five. However, imagine you can just feed it a edited audio clip with the original video and just say like cut my video so that it matches what I did to the and like in the future, probably it's just one whole AI thing. I drop something in, it goes and cuts everything out.”

As this is most probably based upon the Siri voice recognition models, it might be one of the better tools to get a textual form of the episode. Although I understand that enriching this with links and so would be a manual task, this is already suitable to find episodes that discuss a certain topic. Certainly, I helped me to locate the above section of the podcast =).

[source]: https://www.apple.com/newsroom/2024/03/apple-introduces-transcripts-for-apple-podcasts/

**Reply by codereport** [reply]
Yea, i can probably get this with Whisper AI or better. But I have been waiting for time stamped and labeled (with names) transcripts before I start generating them. I also feel like we are a few months away from some website popping up that you just give it a spotify or apple podcast link and it will auto generate a transcript for you.

**Comment by mermerico** [comment]
Thanks for the follow-up, it really clarified a lot of questions I had!

**Comment by phaylon** [comment]
I keep running into this, but I'm not sure where to post it: The ADSP Atom feed (https://adspthepodcast.com/feed.xml) links to the episodes days before they are released. As in, 213 is already there, just without a player loading. That makes it a bit hard to follow along via the feed since I have to keep marking them as unread and then recheck.

**Reply by codereport** [reply]
Ah I didn't realize this. I have been pushing the website the night before on Thursday and then just adding the link. I will stop doing that - sorry for the inconvenience.

**Comment by roibarkan** [comment]
Since Bryce specifically mentioned typos in slides - there's a missing asterisk on the second line of slide 15 of his excellent [execution model talk](https://youtu.be/1-OcraaQl8M?t=380) from meeting C++ 2024.

Keep up the great podcast.
I hope the people on Conor's shortlist know they are on it (and had a chance to improve/change their ways).
Happy 2025

**Comment by robbiesri** [comment]
Thanks for all the clarity in this episode. I was always curious about why y'all didn't discuss some of the negative interactions coming out of the committee and conferences, because I had seen you two be open about your disappointment and frustration on other platforms. I'm grateful that you two spoke out on it. As the kids would say, y'all 'stood on business'.
